services/horizon/docker/docker-compose.yml

```
@@ -21,7 +21,9 @@ services:
       - "11625:11625"
       - "11626:11626"
     env_file:
-      - ./stellar-core-testnet.env      
+      # to connect to the public stellar network
+      # use stellar-core-pubnet.env instead of stellar-core-testnet.env 
```

We could leave the line there ready to be uncommented for them.
```suggestion
      #- ./stellar-core-pubnet.env
```